crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -29,10 +37,11 @@ use rustreexo::accumulator::stump::Stump;
 use sha2::Digest;
 use sha2::Sha512_256;
 
-use super::chainparams::ChainParams;
-use super::error::BlockValidationErrors;
-use super::error::BlockchainError;
-use crate::TransactionError;
+/// The maximum amount of wu's a block can have.
+pub const MAX_BLOCK_WU: u64 = 4_000_000;
+
+/// the static weight of a block header in wu.
+pub const BLOCK_HEADER_WU: u64 = 320;
```

Comment:
Question: Why not move this to `ChainParams`? So we can change this in test scenarios in the future...

Reply:
Good idea